velox/functions/prestosql/tests/StringFunctionsTest.cpp

```diff
@@ -1885,15 +1885,19 @@ TEST_F(StringFunctionsTest, switchCaseCheck) {
     auto result = evaluate<FlatVector<StringView>>(
         query, makeRowVector({conditionVector, argASCII, argUTF8}));
     auto ascii = result->isAscii(rows);
-    ASSERT_EQ(ascii && ascii.value(), isAscii);
+    if (isAscii.has_value()) {
+      ASSERT_EQ(ascii && ascii.value(), isAscii);
+    } else {
+      ASSERT_FALSE(ascii.has_value());
+    }
   };
 
   auto condition = std::vector<bool>{false, true, false};
   auto c1 = std::vector<std::string>{"ali", "ali", "ali"};
   auto c2 = std::vector<std::string>{"àáâãäåæçè", "àáâãäåæçè", "àáâãäå"};
   SelectivityVector rows(condition.size());
   testConditionalPropagation(
-      condition, c1, c2, rows, "if(C0, upper(C1), lower(C1))", true);
+      condition, c1, c2, rows, "if(C0, upper(C1), lower(C1))", std::nullopt);
```

Comment:
What does this change mean? My reading of the original code suggests we were expecting the result to have isAscii flag set, but new code expects the result to not have isAscii flag set. It seems that original expectation is correct.